cli/pcluster/utils.py

```diff
@@ -199,3 +200,41 @@ def get_supported_schedulers():
     :return: a tuple of strings of the supported scheduler
     """
     return "sge", "torque", "slurm", "awsbatch"
+
+
+class NumberedList:
```

Comment:
Not clear what's the real value added by this class over a simple list other than accessing values starting from the index 1

Reply:
The real values it that given a list, it adds a number to each element and allows to retrieve each one, knowing the number, in O(1). It also returns the list with the numbers as an ordered dict, making it easier to print for the user. It's not a very complicated class, that's for sure. I think that in this way is more organized